CMakeLists.txt

```diff
@@ -130,12 +124,15 @@ set(COMMON_CXX_FLAGS
 )
 
 # Option for enabling/disabling SIMD flags is for both of debug and release
-if (QUADIRON_USE_SIMD STREQUAL "SSE4")
+if (USE_SIMD STREQUAL "ON")
+  SET(COMMON_CXX_FLAGS "${COMMON_CXX_FLAGS} -march=native")
```

Comment:
Last time we do agree to not using `-march=native` ;) https://github.com/scality/quadiron/pull/190#discussion_r172463397

Reply:
Yup, I said "IMHO, if we want to compile with march=native it should be done by enabling an option." and it's the case here (it's not always enabled, only when USE_SIMD is ON).